.gitignore

```diff
@@ -3,3 +3,4 @@ pubspec.lock
 packages
 .pub
 .packages
+.idea
```

Comment:
Please don't add IDE-specific ignores! See https://www.dartlang.org/guides/libraries/private-files

Reply:
Oh sorry! Didn't know that was a guideline.

Reply:
Mh, on the page you linked there is no mention of not adding IDE-specific ignores (only that one shouldn't actually commit IDE-specific files).

Reply:
(I think Kevin was referring to "To avoid committing files that are specific to your personal workflow or setup, consider using a global ignore file (for example, .gitignore_global)".)

Though I don't mind having workflow specific files in `.gitignore`. It's better than someone accidentally committing editor specific things into the repository.